best-practices/gke-batch-refarch/gke/main.tf

```diff
@@ -21,121 +21,15 @@ locals {
 
 data "google_client_config" "default" {}
 
-provider "kubernetes" {
-  host                   = "https://${resource.google_container_cluster.gke_batch.endpoint}"
-  token                  = data.google_client_config.default.access_token
-  cluster_ca_certificate = base64decode(google_container_cluster.gke_batch.ca_certificate)
+data "google_container_cluster" "gke_cluster" {
+  name     = "batch-dev"
```

Comment:
Can we use a variable defined in a tfvars file instead ? And maybe use a prefix for different envs ? Like dev-, staging- ?

Reply:
I think there are several enhancements we can make, but I think Ali's idea was to keep things as simple as possible for the tutorial. Maybe we can add a guide and additional configuration for customizing the deployment.